app/services/provider_interface/hesa_data_export.rb

```diff
@@ -2,12 +2,15 @@ module ProviderInterface
   class HesaDataExport
     NO_INFORMATION_GIVEN_STRING = 'no information shared'.freeze
 
-    def initialize(actor:)
+    attr_reader :actor, :recruitment_cycle_year
+
+    def initialize(actor:, recruitment_cycle_year: RecruitmentCycle.current_year)
```

Comment:
Are there any implicit dependencies on current_year in this file? Some of the mappings might depend on the year?

Reply:
Not as far as I'm aware of. The report appears to be generated with data relevant to the year that is specified as well.